.circleci/config.yml

```diff
@@ -639,23 +653,41 @@ workflows:
             branches:
               only: env/dev2
 
-      - hodl:
+      - hodl_blue:
           type: approval
           filters:
             branches:
               ignore: /.*/
             tags:
               only: /^v.*$/
 
-      - deploy_uat:
+      - hodl_green:
+          type: approval
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v.*$/
+
+      - deploy_uat_blue:
           requires:
             - eunit
             - static_analysis
             - linux_package
-            - hodl
+            - hodl_blue
           filters:
             branches:
-              ignore: /.*/
+              only: env/uat_blue
+            tags:
+              only: /^v.*$/
+
+      - deploy_uat_green:
+          requires:
+            - deploy_uat_blue
+            - hodl_green
```

Comment:
Let us say that, on `v.*` tag:
* Job `deploy_uat_blue` succeeds;
* `hodl_green` is not complete (clicked...) yet;
* Validating blue deployment, validating pushed Docker image, validating uploaded macOS package, take some time (so `hodl_green` not clicked yet).

Does `hodl_green` and the rest of the workflow not time out in - let us say - 3 days? (Checking what the workflow is meant to be.)

----

A related question is whether such a partially-completed deployment shall be completed by pushing branch `env/uat_green` (and whether that works - see comment below).

Reply:
> Does hodl_green and the rest of the workflow not time out in - let us say - 3 days? (Checking what the workflow is meant to be.)

I sent the question to CircleCI support as I can't find it documented.

Reply:
> A related question is whether such a partially-completed deployment shall be completed by pushing branch env/uat_green (and whether that works - see comment below).

Partially-completed deployment should be retried with workflow retry.

Reply:
> I sent the question to CircleCI support as I can't find it documented.

Just received an reply, we're all good. We can leave it waiting for a week or so. The timeout is only on job execution.

Reply:
> Partially-completed deployment should be retried with workflow retry.

Workflow retry or [job retry](https://github.com/aeternity/epoch/wiki/Release-process/_compare/c953b52b0f223797fcdc4941a75318edee4e27f1...74f96f819fd263886ad98d6c1e75a06cc5d8abc6#diff-991db8edfb34dd9dafc5302987e97019R28)?

Reply:
Yes, job retry.